webapp/frontend/lib/settings/settings_service.dart

```diff
@@ -20,4 +23,22 @@ class SettingsService {
       return "unknown";
     }
   }
+
+  Future<List<String>> getSeedPhrase() async {
+    // TODO(holzeis): this should come from the config
+    const port = "3001";
+    const host = "localhost";
+
+    try {
+      final response = await http.get(Uri.http('$host:$port', '/api/seed'));
```

Comment:
Check out this client: 

https://github.com/get10101/10101/blob/dd57e3e6ff645685267929440c50daf9a86b9f40/webapp/frontend/lib/common/http_client.dart#L12-L11

```suggestion
      final response = await HttpClientManager.instance.get(Uri.http('$host:$port', '/api/seed'));
```